action-scheduler.php

```diff
@@ -40,14 +40,7 @@ function action_scheduler_register_3_dot_1_dot_0() {
 	}
 
 	function action_scheduler_initialize_3_dot_1_dot_0() {
```

Comment:
Does this method name need to be changed at all to match the next version? 3.1.1?

Reply:
> Does this method name need to be changed at all to match the next version? 3.1.1?

Yes, we do the version bump in its own PR #473.